### ISSUE TITLE
Format error message should include the regex

### ISSUE THREAD

**Issue by blacksails** [CargoSense/vex#23]
Consider the following validations for a password:
```elixir
password: [presence: true,
           length: [min: 8],
           format: ~r/[0-9]/,
           format: ~r/[a-z]/,
           format: ~r/[A-Z]/]
```

It would be nice to be able to tell which of the formats are invalid.

**Comment by bruce** [comment]
The default messages are generally meant to be visible to the end user (and I'd argue regular expressions fail that test) _but_ I'm willing to make the pattern available for custom error message handling, eg, adding `pattern` to https://github.com/CargoSense/vex/blob/master/lib/vex/validators/format.ex#L43

**Comment by blacksails** [comment]
Okay sorry I didn't know that the default messages were meant for end users, in that case I completely agree with you.

But I would still like to be able to convert the different format errors to an appropriate error message for my end users. I am very new to the vex library, just spent an hour with it this evening and found that I missed this feature. Also I don't know the code base. Therefore I dont really know what would be the best solution.

**Comment by bruce** [comment]
No problem. In short, any validation can be provided a custom message format to use. I just need to add the ability for the pattern to be available; I'll post an example once it's ready.

**Comment by blacksails** [comment]
Okay that really sounds cool :) Thanks for a lot your effort!

**Comment by djspinmonkey** [comment]
I think this is fixed by #35.

**Comment by djspinmonkey** [comment]
Released in v0.6.0
